ui/vendor/css/bootstrap.css

```diff
@@ -910,12 +1097,12 @@ textarea {
   line-height: inherit;
 }
 a {
-  color: #428bca;
+  color: #337ab7;
```

Comment:
So they've also changed link colors everywhere, if we care about that. Darker and less saturated blue.

![links](https://cloud.githubusercontent.com/assets/3660661/8853981/7567966c-312b-11e5-9143-9163de0595eb.jpg)